src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -139,6 +143,12 @@ namespace AppInstaller::CLI::Workflow
 
     void VerifyInstallerHash(Execution::Context& context)
     {
+        const auto& installer = context.Get<Execution::Data::Installer>().value();
+        if (installer.InstallerType == ManifestInstaller::InstallerTypeEnum::MSStore)
+        {
+            return;
+        }

```

Comment:
I would rather change how this is decided to be run than to start building early outs into every function.  Probably better to move calling it out of the base flow and into DownloadInstaller where it is needed.